src/components/Button/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styles from "./styles/styles.module.scss";
+
+const Button = ({ title, path }) => {
+  let buttonStyle = styles.primaryButton;
+
+  if (title === "Confirm") {
+    buttonStyle = styles.confirmButton;
+  }
+
+  return (
+    <Link to={path}>
+      <button className={buttonStyle}>{title}</button>
```

Comment:
If you apply the comment on `src/components/Button/styles/styles.module.scss` this would probably change.

Reply:
yessss